src/Symfony/Component/Translation/Command/TranslationPushCommand.php

```diff
@@ -133,7 +132,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $localTranslations = $this->readLocalTranslations($locales, $domains, $this->transPaths);
 
         if (!$domains) {
-            $domains = $this->getDomainsFromTranslatorBag($localTranslations);
+            $domains = $provider->getDomains();
```

Comment:
We need to check if `$provider` has `getDomains` method here, as it's done here https://github.com/symfony/symfony/pull/45171/files#diff-a00f86baf85c337ee61148e5fff8d189edf224ef40382518307359f579f40e6aR136